log_collector.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
sadly, not everyone is using python3

Reply:
One of the major points of this PR (and subsequent work) was to get this code Python 3 ready.

Reply:
Not sure we should assume python3 is available on all instances which run this script.